analyzer/src/main/kotlin/managers/GoDep.kt

```diff
@@ -253,7 +253,13 @@ class GoDep(
     }
 
     private fun resolveVcsInfo(importPath: String, revision: String, gopath: File): VcsInfo {
-        val pc = ProcessCapture("go", "get", "-d", importPath, environment = mapOf("GOPATH" to gopath.path))
+        val pc = ProcessCapture(
+            "go",
+            "get",
+            "-d",
+            importPath,
```

Comment:
Nit: Even if it somehow violates our formatting habits, maybe put all of

    "go", "get", "-d", importPath

in a single line, as it all belongs to the same CLI call?